testdata/1_input

```diff
@@ -0,0 +1,8 @@
+pick 4365d5d [DROP][INVESTIGATE] Weird shell diffs
```

Comment:
I can add comments on top of these files - basically 1 is the example you had in Trello, 2 and 3 are common scenarios and 4+5 are testing SQUASHes before their CARRY counterparts (you asked for that:) ).